exporters/inmemory/src/test/java/io/opentelemetry/exporters/inmemory/InMemorySpanExporterTest.java

```diff
@@ -103,12 +107,12 @@ public void export_ReturnCode() {
   static SpanData makeBasicSpan() {
     return TestSpanData.newBuilder()
         .setHasEnded(true)
-        .setTraceId(io.opentelemetry.trace.TraceId.getInvalid())
-        .setSpanId(io.opentelemetry.trace.SpanId.getInvalid())
+        .setTraceId(TraceId.getInvalid())
```

Comment:
Is this style change (use an import instead of the full path for a class) imposed by this errorprone update?

*Personally*, when a class is used only once in a file (such as `SpanId` here), I see no problem using its full path ;)

Reply:
yep..and this is one of the errorprone warnings that I actually agree with. I don't think the "never import inner classes" is all that great.

Reply:
also, sounds like you need to join the real IDE generation, @carlosalberto .  ;)